configure.ac

```diff
@@ -613,6 +613,16 @@ test -n "$opt_kerneloverflowuid" ||
           opt_kerneloverflowuid=65534
 AC_DEFINE_UNQUOTED(PAM_USERTYPE_OVERFLOW_UID, $opt_kerneloverflowuid, [Kernel overflow uid.])
 
+AC_ARG_WITH([systemduserunitdir],
```

Comment:
Actually this is still wrong - this should not be a systemd user unit dir but systemd system unit dir. ${prefix}/lib/systemd/system
And everything needs to be renamed accordingly.

Reply:
Thanks.  Updated according to the comments.